### ISSUE TITLE
How to prevent OnSelectedItemChanged Call after Refresh

### ISSUE THREAD

**Issue by minas15** [ant-design-blazor/ant-design-blazor#2046]
Hi,

Each time the page is reloaded or clicked, the OnSelectedItemChanged for Select is executed. How can I prevent this? The same behavior can be seen on the Ant Blazor Page. The OnSelectedItemChanged should only be called, if I select an Item

 ```
<Select @bind-Value="@context.ColorState" 
		DefaultValue = "@context.ColorState"
		TItemValue="ColorState" 
		TItem="ColorState"
		OnSelectedItemChanged="(value)=>UpdateColor(context)">
	<SelectOptions>
		@foreach (ColorState color in Enum.GetValues(typeof(ColorState)))
		{
			<SelectOption TItemValue="ColorState" TItem="ColorState" Value="@color" />
		}   
	</SelectOptions>
</Select>  
```

**Comment by anddrzejb** [comment]
@anranruye thank you for sharing the issue. It was a worthwhile read. And I agree with your understanding of properties use when marked with `[Parameter]` attribute. Although I am not sure if the suggestion of forcing only `{get;set;}` is entirely warranted. Of course, without understanding of possible underlying problems, it is a good solution, but in the hands of a programmer well versed in blazor internals, using setters (or getters) can help keeping the code cleaner & leaner. 

But now I can see that there was a misunderstanding (or at least I think there was). I was not referring to changing value of the child component (so the `[Parameter] Value {get;set;}`, I was rather referring to changing the parent's variable that is bound to `Value` parameter. But the more thought I gave to the issue, the more I think @susahin80 is actually correct.

**Comment by JKoffeman** [comment]
I agree with @susahin80 that OnSelectedItemChanged should only be called when user actually changes the value.

**Comment by ElderJames** [comment]
Hello @JKoffeman , did you met this issue with latest version?
